lib/src/geo/crs.dart

```diff
@@ -2,6 +2,7 @@ import 'dart:math' as math hide Point;
 import 'dart:math' show Point;
 
 import 'package:flutter_map/src/misc/bounds.dart';
+import 'package:flutter_map/src/misc/simplify.dart';
```

Comment:
(for the future: if we're starting to migrate to `DoublePoint` throughout (which I agree with), we should probably separate it into its own file, and make the internal changes all at once as much as possible. Currently, I think we're split between `Offset`, `Point<double>`, `DoublePoint` and records - they are all used in different places obviously, but we should try to get just one or two (eg. `Offset` cannot be avoided in the painter).)

Reply:
I will handle this in #1996